core/src/main/java/tech/tablesaw/io/csv/CsvWriteOptions.java

```diff
@@ -159,16 +166,37 @@ protected Builder(OutputStream stream) {
       return this;
     }
 
+    /**
+     * Writes date column output using the given DateFormatter
+     *
+     * @deprecated
+     */
+    @Deprecated
     public CsvWriteOptions.Builder dateFormatter(DateTimeFormatter dateFormatter) {
       this.dateFormatter = dateFormatter;
       return this;
     }
 
+    /**
+     * Writes DateTime column output using the given DateFormatter
+     *
+     * @deprecated
+     */
+    @Deprecated
     public CsvWriteOptions.Builder dateTimeFormatter(DateTimeFormatter dateTimeFormatter) {
       this.dateTimeFormatter = dateTimeFormatter;
       return this;
     }
 
+    /**
+     * Sets the usePrintFormatters option @see{ColumnPrintFormatter} When true, printFormatters will
```

Comment:
I've never seen `@see` used like this. I think the standard way to do this might be `{@link ColumnPrintFormatter}`